src/services/panelService.ts

```diff
@@ -93,9 +93,9 @@ const getPanelBasedOnPanelSetAndIndex = (sequelize : Sequelize) => async (index
  * @returns {object[]} An array of objects with id, image, index properties
  */
 const getPanelsFromPanelSetID = (sequelize : Sequelize) => async (panel_set_id: number) => {
-
     // Find all panels on requested panelSet 
-    const panels = await sequelize.models.panel.findAll({ where: { panel_set_id } }) as IPanel[];
+    const panel_set = await sequelize.models.panel_set.findByPk(panel_set_id, {include: sequelize.models.panel}) as IPanelSet;
+    const panels = panel_set.panels as IPanel[];
```

Comment:
This should probably be changed to return an empty array, and return a 404
![image](https://github.com/RIT-Crowd-Comic/crowd-comic-back-end/assets/62062830/36652f83-4a39-4ac6-8196-fb6b4092a18b)